collector/runit.go

```diff
@@ -54,6 +54,16 @@ func NewRunitCollector() (Collector, error) {
 			},
 			labelNames,
 		),
+		stateTimestamp: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace:   Namespace,
+				Subsystem:   subsystem,
+				Name:        "state_last_change",
+				Help:        "the time of the last runit service state change.",
```

Comment:
Style nit: Please start help strings capitalized, end with period. And maybe be very clear about what "time" means, like:

`Unix timestamp in seconds of the last runit service state change.`